libiocage/lib/Host.py

```diff
@@ -21,9 +21,22 @@ def __init__(self, root_dataset=None, zfs=None, logger=None):
             logger=self.logger
         )
 
+        self._devfs = None
         self.releases_dataset = None
 
     @property
+    def devfs(self):
+        """
+        Lazy-loaded DevfsRules instance
+        """
+        if self._devfs is None:
+            import libiocage.lib.DevfsRules
```

Comment:
We should import at the top, this is against PEP8 (https://www.python.org/dev/peps/pep-0008/#imports)

Reply:
Do you know a way to still lazy load the module? It's not required most of the scenarios using libiocage, so I thought delaying the loading can be benifitial for all other tasks. The performance penalty will probably not be noticeable, so in case you don't know a way, I'm up to move it to the files header.

Reply:
I can't think of one offhand, but yeah the penalty will be so minimal. It's nice to have all our imports in one place.